stable/secrets/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+Secrets chart is used with conjunction of other charts. It can come in handy when a public chart does not provide support for additional secrets. In that scenario, this chart can be used to deploy additional secrets.
```

Comment:
NOTES.txt is intended for post install messages. It is not just a restatement of the purpose of the chart.

Reply:
Updated